framework_tvb/tvb/interfaces/web/controllers/flow_controller.py

```diff
@@ -489,7 +489,7 @@ def execute_post(self, project_id, submit_url, step_key, algorithm, **data):
             common.set_important_message(str(result))
 
         except formencode.Invalid as excep:
-            errors = excep.unpack_errors()
+            errors = excep.unpack_errors
```

Comment:
I do not think this solves the problem. Look at the printed message.
Like this, we say that variable `errors` is the reference towards a function. It might "work", meaning that it prints something and does not throw an error, like it did before, but it does not say what is the 'required' missing attribute (or whatever the specific exception might be in the form validation).

Reply:
The printed message looks like that:
2020-02-26 14:05:20,413 - WARNING - tvb.interfaces.web.controllers.flow_controller - Invalid form inputs <bound method Invalid.unpack_errors of Invalid('Could not fill algorithm from the given inputs!', {}, None, None, {'_time_series': ValueError('Field required'), '_dt': ValueError('Field required'), '_neural_input_transformation': ValueError('Field required'), '_bold_model': ValueError('Field required')})>

Reply:
yep. I am right, it is the __str__ on the function `<bound method .... of ...[class]...`
It shows un indeed the problem, but it is a hack